### PR TITLE
RFC: Remove authors' name

### LINKED ISSUE #394: remove authors' names
I propose we remove authors' names from all files in line with standard open source conventions.

Example: https://github.com/QuantEcon/QuantEcon.py/blob/master/quantecon/discrete_rv.py

It might be worth flagging this on QuantEcon.jl

### PR DESCRIPTION
**Pull request by QBatista**
Close #394 

The format of the description is not consistent across files. Here's my proposal to make it consistent:

```
"""
Filename: <filename>

<Description>

References (if applicable)
----------
 <References> 
"""
```

### CODE REVIEW

**Comment by oyamad** [comment]
I would say we should also remove the `Filename` entry, as well as `Date` and `@date`.

For example: http://quanteconpy.readthedocs.io/en/latest/game_theory/random.html
"Filename: random.py" adds no information given the header "**random**".

**Comment by QBatista** [comment]
That's fine for me too.

**Comment by jstac** [comment]
Thanks @QBatista, much appreciated.
@oyamad I agree

File: quantecon/util/timing.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
 """
 Filename: timing.py
```
Comment (mmcky) [inline quantecon/util/timing.py]: this could be deleted also.

File: quantecon/util/timing.py [hunk 2]
```diff
@@ -1,8 +1,6 @@
...
 
-Authors: Pablo Winant
-
 Date: 10/16/14
```
Comment (mmcky) [inline quantecon/util/timing.py]: and this

Comment (QBatista) [reply quantecon/util/timing.py]: Done, thanks!

**Comment by mmcky** [comment]
thanks @QBatista when you're happy with this PR can you attach the `ready` label?

**Comment by QBatista** [comment]
@mmcky I think I don't have access to label PRs yet

**Comment by mmcky** [comment]
oh - I didn't realize that wasn't accessible. I have added the label and will merge tomorrow. I usually leave it a day or so at ``ready`` status.

**Pull request merged**